rules/judgment.tex

```diff
@@ -84,3 +84,8 @@ \subsection{Referees During the Match}
 The head referee and the assistant referees should wear socks \emph{of black or dark blue color} and avoid reserved colors (white and green) in their leg clothing.
 They may enter the field in particular situations, \eg, to remove a robot when applying a penalty.
 They should avoid interfering with the robots as much as possible.
+
+\subsubsection{Visual signal}
+
+For Challenge Shield matches only, the referee executing the signal will wear red gloves to increase visibility; however, teams are advised not to rely solely on this visual cue, as gloves may be omitted in future years.
+No other objects or products covering the hands are allowed; any exception must be approved by the Technical Committee.
```

Comment:
This would imply e.g. no rings or jewlery? 
I am also not sure if we need another subsection here? Shouldn't this in any case be at the CS specific differences?